packages/core/src/util/config.ts

```
@@ -16,6 +16,10 @@ export class Config {
     return process.env.NODE_ENV === this.PROD_ENV;
   }
 
+  static isProduction(): boolean {
+    return Config.getEnvType() === this.PROD_ENV;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)